resources/asciidoctor/spec/elastic_compat_preprocessor_spec.rb

```diff
@@ -245,31 +245,21 @@
     expect(actual).to eq(expected.strip)
   end
 
-  it "attribute only blocks don't pick up blocks with attributes and other stuff" do
-    actual = convert <<~ASCIIDOC
-      == Header
-
-      --
-      :attr: test
-      added[some_version]
-      --
-
-      [id="{attr}"]
-      == Header
-    ASCIIDOC
-    expected = <<~DOCBOOK
-      <chapter id="_header">
-      <title>Header</title>
-      <note revisionflag="added" revision="some_version">
-      <simpara></simpara>
-      </note>
-      </chapter>
-      <chapter id="test">
-      <title>Header</title>
+  context 'when a block contains things other than attributes' do
```

Comment:
I reworked this one rather than just chopping the line because I'd forgotten the goal.